### ISSUE TITLE
Russian text hide after export as pdf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [umlet/umlet#51]
```
Steps to reproduce
1. Create any diagram with russian text
2. Export it as pdf

Expected: Correct PDF
Instead: PDF without russian text

UMLet version: 11.3
OS: Gentoo Linux or Windows 7
Java: jre update 23(on windows), jre from ibm on linux

I attached sample uxf-file to reproduce this issue
```

Original issue reported on code.google.com by `KultiH...@gmail.com` on 8 Sep 2011 at 7:34

Attachments:
* [new.uxf](https://storage.googleapis.com/google-code-attachments/umlet/issue-51/comment-0/new.uxf)

**Comment by afdia** [comment]
a similar issue is #162

**Comment by afdia** [comment]
The bold/italic issue has been fixed in #195 by allowing the user to specify separate fonts (there does not seem to be an easier way to fix this)

**Comment by afdia** [comment]
with the addition of GUI support for export fonts I'm closing this issue, because there is no intuitive alternative way to solve the problem.
The user will have to get an UTF8 font from somewhere and has to set it in the options
